core/src/main/java/org/mule/runtime/core/processor/AbstractFilteringMessageProcessor.java

```diff
@@ -27,6 +31,9 @@
  */
 public abstract class AbstractFilteringMessageProcessor extends AbstractInterceptingMessageProcessor {
 
+  public static final String FILTER_ON_UNACCEPTED_NOT_STOP_PARENT_FLOW =
```

Comment:
This flag was added to keep backwards compatibility.
Do you prefer remove it and to take this behaviour as the default and unique in mule-4.x?